output/schema/schema.json

```diff
@@ -111810,7 +111810,7 @@
       ],
       "query": [
         {
-          "description": "Specifies what to do when the request:\n- Contains wildcard expressions and there are no models that match.\n- Contains the _all string or no identifiers and there are no matches.\n- Contains wildcard expressions and there are only partial matches.",
+          "description": "Specifies what to do when the request: - Contains wildcard expressions and there are no models that match. - Contains the _all string or no identifiers and there are no matches. - Contains wildcard expressions and there are only partial matches.",
```

Comment:
Why don't we allow line breaks in descriptions? The above was a list and will be output as a weird single line with dashes.

We should settle on a markup language for descriptions that allows formatting for long descriptions. Considering the use of lists and `` `code` `` snippets, I would suggest CommonMark that has implementations in most languages and can thus be used in code generators.